tools/checkTheme.js

```diff
@@ -167,37 +167,36 @@ function check_group(group, rules) {
     return [scope, has_rule(selector, rules), skips_rule(selector, rules)];
   });
 
-
   const doesNotSupport = has_rules.map(x => x[1]).includes(false);
   const skipped = has_rules.find(x => x[2]);
   if (doesNotSupport || skipped) {
-    console.log(group.name.yellow);
+    console.log(yellow(group.name));
     if (doesNotSupport) {
-      console.log(`- Theme does not fully support.`.brightMagenta);
+      console.log(magentaBright`- Theme does not fully support.`);
```

Comment:
with `ansis` you can use the template string syntax very well, 
this is clear and readable code unlike using extended `String.prototype` by `@colors/colors`